ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.10.yaml

```diff
@@ -39,7 +39,7 @@ tests:
     cluster_profile: aws-qe
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
-    workflow: cucushift-installer-rehearse-aws-ipi-ovn-fips
+    workflow: cucushift-installer-rehearse-aws-ipi-capi
```

Comment:
@miyadav You should add your workflow in https://github.com/openshift/release/blob/master/ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.11.yaml to test, as capi works starting from 4.11